http-python3.12-flask3.0/requirements.txt

```diff
@@ -0,0 +1 @@
+flask
```

Comment:
If we want to tie this example to a specific flask semver (as the dirname suggests), we should use version specifiers here:
```
flask>=3.0,<3.1
```